lib/web/apiserver.go

```diff
@@ -873,6 +873,7 @@ func (h *Handler) bindDefaultEndpoints() {
 
 	// Audit events handlers.
 	h.GET("/webapi/sites/:site/events/search", h.WithClusterAuth(h.clusterSearchEvents))                 // search site events
+	h.GET("/v2/webapi/sites/:site/events/search", h.WithClusterAuth(h.clusterSearchEventsV2))            // search site events
```

Comment:
i would just add a note why v2 is different, like supports search?

and does the v1 need to be deprecated? (the // TODO: DELETE)